changelog.md

```diff
@@ -1,3 +1,6 @@
+## 0.9.0 [unreleased]
```

Comment:
Can you also bump the version number in `wai-middleware-static` as well?

Reply:
Done.